src/app/profile/yourGiving/editRecurringGifts/step1/editRecurringGifts.tpl.html

```diff
@@ -25,9 +25,9 @@ <h3 translate>Make changes to your recurring gifts</h3>
         <div class="alert alert-warning" role="alert" ng-if="!$ctrl.loading && !$ctrl.loadingError && (!$ctrl.recurringGifts || $ctrl.recurringGifts.length === 0)">
           <p translate>You don't seem to have any recurring gifts to edit. You may click continue to set up a new recurring gift for a person or ministry you have given to recently.</p>
         </div>
-        <div ng-repeat="gift in $ctrl.recurringGifts" class="row repeating-row user-profile">
-          <gift-list-item gift="gift">
-            <gift-update-view gift="gift"></gift-update-view>
+        <div ng-repeat="recurringGift in $ctrl.recurringGifts" class="row repeating-row user-profile">
+          <gift-list-item gift="recurringGift">
+            <gift-update-view gift="recurringGift.gift"></gift-update-view>
```

Comment:
This needs to be using the original object as we are using https://github.com/CruGlobal/give-web/blob/master/src/common/models/recurringGift.model.js which stores the updated value in a different location than the original value. I can look at this more later today. I think we're going to have to modify my detection of if the amount has changed in the recurringGift model or not save it until it is valid.